src/components/CtaButton.js

```diff
@@ -3,10 +3,13 @@ import styled from 'styled-components';
 
 const CtaStyle = styled.button`
   padding: 12px;
+  width: 172px;
+  height: 64px;
+  align-items: center;
```

Comment:
Does this stuff do anything? Can't see if CtaStyle is flex?

Reply:
ye that stuff makes the button that size, and makes the text and chevron aligned in the center